src/main/resources/aurora-openshift-spring-boot-starter.properties

```diff
@@ -1,6 +1,7 @@
 management.health.status.order=DOWN, OUT_OF_SERVICE, UNKNOWN, OBSERVE, UP
 management.health.status.http-mapping.OBSERVE=200
 management.server.port=${MANAGEMENT_HTTP_PORT:8081}
+management.server.web.base-path=/
```

Comment:
Jeg mener property som skal settes er

management.endpoints.web.base-path=/

Reply:
Så fint at dette engasjerer!

Nå var ikke jeg klar over at det var noe verdi i selve actuator pathen. om det er det så kan vi jo søke å bevare de egenskapene. I og med at vi her har management på en annen port enn applikasjonen kjører på - kan vi si at port 8081 er actuator? Gitt at man ikke får 404 på / da.

Ellers så er jeg med på at vi også kan endre vår standard for åetterkomme spring sine standarder. 
Her kan jo et alternativ være om platformen kan lete etter flere URL'er. F.eks. localhost:8081/actuator/prometheus eller localhost:8081/prometheus ?